superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx

```diff
@@ -320,7 +323,7 @@ const FilterBar: React.FC<FiltersBarProps> = ({
             activeKey={editFilterSetId ? TabIds.AllFilters : undefined}
           >
             <Tabs.TabPane
-              tab={t(`All Filters (${filterValues.length})`)}
+              tab={`${t('All filters')} (${numberOfFilters})`}
```

Comment:
i know you didn't add this (and instead are just touching it to update the casing of the string), but this translation string isn't correct. you need something like:
```
t('All filters (%(numberOfFilters))', numberOfFilters)
```

see here for more details: https://github.com/apache-superset/superset-ui/tree/master/packages/superset-ui-core/src/translation#api

you'll also want to change the one on line 346